src/components/Pages/Home.js

```diff
@@ -5,7 +5,7 @@ import { injectIntl } from 'react-intl';
 import { misc as i18n2, services as i18n3 } from 'js/i18n/definitions';
 
 import WorkInProgress from 'components/WorkInProgress';
-import PageSubBanner from 'components/PageSubBanner';
+import PageNotificationBanner from 'components/PageNotificationBanner';
```

Comment:
Talked to @cthibaultatx  about the rename and we landed on something between banner and what we originally labelled (PageNotification) in our pin up.